Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -101,8 +101,9 @@ To create a **HoloLens application**, switch to the Universal Windows Platform:
     Profiles configure the behavior of MRTK core components and are described in more detail in the [profiles](Profiles/Profiles.md) article.
 
     > [!NOTE]
-    > If you are getting started on the HoloLens or HoloLens 2, you should choose the "DefaultHoloLens1ConfigurationProfile" or DefaultHoloLens2ConfigurationProfile" instead.
-    > See the [profiles](Profiles/Profiles.md#hololens-2-profile) for more information on the differences between DefaultMixedRealityToolkitConfigurationProfile and DefaultHoloLens2ConfigurationProfile.
+    > * If you're using Unity's XR SDK in Unity 2019.3 or later, you should choose the "DefaultXRSDKConfigurationProfile". This profile is set up with MRTK's XR SDK systems and providers, where needed.  
```

Comment:
What are your thoughts on if we should filter out certain profiles from the list based on the current Unity version? i.e. so we can prevent people from doing the wrong thing

Reply:
Different PR obviously

Reply:
Yes! I've had that same thought in the back of my head for a bit.
I think it makes sense, since, as far as Unity is concerned, the two XR paths aren't really mix-and-match. It's one or the other, and only one once we hit Unity 2020.